Hadrons/Modules/MGuesser/BatchExactDeflation.hpp

```diff
@@ -83,36 +84,87 @@ class BatchExactDeflationGuesser: public LinearFunction<typename Pack::Field>
         unsigned int sourceSize = out.size();
 
         LOG(Message) << "=== BATCH DEFLATION GUESSER START" << std::endl;
+        LOG(Message) << "Total Sources: " << sourceSize << std::endl;
```

Comment:
The timer logic here is quite messy and could be done in a much smaller amount of code I think. Hadrons has a TimerArray class exactly to streamline that kind of timing.

Reply:
timers changed to Hadrons ones as recommended.